pyproject.toml

```diff
@@ -17,7 +17,7 @@ dependencies = [
     "cohere",
     "elasticsearch >= 8.5",
     "elasticsearch-dsl",
-    "fastapi",
+    "fastapi <= 0.112.0",
```

Comment:
Newer versions are incompatible with `fastapi_pagination` waiting for a fix on either side to unpin.